CONTRIBUTING_DOCS.md

```diff
@@ -54,18 +53,27 @@ make clean         - Removes the local `public` directory, which is the default
 
 ## Add new documentation
 
-We provide template files for different types of documentation. The templates, including instructions to use them and examples, are located in the [templates](templates) directory.
+We use [Hugo archetypes](https://gohugo.io/content-management/archetypes/) to provide structure for new documentation pages.
+
+These archetypes include inline advice on Markdown formatting and our most common style guide conventions.
+
+To create a new page, run the following command:
+
+`hugo new content <product/folder/filename.md>`
+
+This new page will be created with the default how-to archetype. To use a specific archetype, add the `-k` parameter and its name, such as:
+
+`hugo new content <product/folder/filename.md> -k <archetype>`
+
+Our archetypes [currently include](/archetypes/) the following:
```

Comment:
Why are we leaving out the other templates?

Reply:
They have not been maintained or used.

Reply:
```suggestion
Our templates [currently include](/archetypes/) the following:
```

Reply:
> They might still be adapted for use in archetypes in the future

Are you saying that we can't use these other templates with the `hugo new content` command?

Reply:
> Are you saying that we can't use these other templates with the hugo new content command?

Basically, yeah. The templates do not interact in any meaningful way with Hugo archetypes.

There's a minimal amount of work needed to "convert" the templates into Hugo archetypes, but the bigger issue is that the templates were worked on in a vacuum and never adopted, whereas the archetypes were rewrites of templates to reflect the actual content IA patterns in production documentation.

I would personally like to remove the files, but it feels beyond the scope of the PR, though the fact nobody has interacted with them since creation means it's likely fine.